contract-message-script/src/transaction_utils.sw

```diff
@@ -0,0 +1,15 @@
+library transaction_utils;
```

Comment:
Are there other types of utils that I haven't seen yet and must be differentiated? Perhaps `utils` on its own is sufficient?

Reply:
It was made more specific because there are some utils functions defined in the main sway contracts themselves. These files are specific to util functions that relate to the GTF opcode.